roles/install_yamls/README.md

```diff
@@ -57,3 +57,66 @@ The role can be imported and tasks can be executed like this
 
 Tasks are created in a form of: `make_{{ original make target }}`, corresponding to `make target` imported from [a Makefile](https://github.com/openstack-k8s-operators/install_yamls/blob/c8487df41bf9ddefa7989f9384e77ae9720ce9dd/Makefile#L418).
 When task is being executed, it runs corresponding code from the Makefile.
+
+## Make install_yaml Parameter
+
+### module: generate_make_tasks
+
+1. For every makefile target we have a dedicated task file.
+
+```{collapse} Show code
+    ~~~{literalinclude}
```

Comment:
no need to collapse here.
Also, literalinclude isn't needed either, and would crash the doc compilation anyway.

You want a simple:
```
```YAML
```

Reply:
done